tools/elastic/migration/MigrationBuilder.cs

```diff
@@ -16,6 +17,11 @@ public class MigrationBuilder
     /// </summary>
     public IElasticClient Client { get; }
 
+    /// <summary>
+    /// get - Elastic client for indexing because the IElasticClient does not use System.Text.Json.
+    /// </summary>
+    public ElasticsearchClient IndexingClient { get; }
```

Comment:
It is annoying to have two elastic clients, but to rewrite all the functionality that came with NEST will take far too long.